app/services/actions/foreman/puppet_class/import.rb

```diff
@@ -0,0 +1,30 @@
+module Actions
+  module Foreman
+    module PuppetClass
+      class Import < Actions::EntryAction
+        def resource_locks
+          :import_puppetclasses
+        end
+
+        def run
+          output[:changed] = input[:changed]
```

Comment:
oh, and also to avoid `{"results"=>[]}` default output I'm getting otherwise, not entirely sure where it comes from